src/context/useGetDomainMetadata.ts

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from "react"
+import { Actions, EnvironmentMetadata, ResourceMetadata } from "../types"
+
+export function useGetSharedConfig(explorerContextDispatcher: React.Dispatch<Actions>): void {
```

Comment:
```suggestion
export function useGetDomainMetadata(explorerContextDispatcher: React.Dispatch<Actions>): void {
```

You can name it some other way, but for sure it should be changed from shared config reference

Reply:
I was pretty sure I renamed but perhaps I forgot to push this change